### PR TITLE
Implement configuration of Aurora basic authentication

### PR DESCRIPTION
**Pull request by FlorentFlament**

### CODE REVIEW

File: templates/default/aurora-security.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[users]
+<% @admins.sort.each do |key, value| %>
+<%= key %> = <%= value %>, admin
+<% end %>
+
+[roles]
+admin = *
```
Comment (pierrecdn) [inline templates/default/aurora-security.erb:7]: OPT: roles might be configurable (according to http://aurora.apache.org/documentation/latest/operations/security/#using-an-ini-file-to-define-security-controls, we can set different types of roles to the groups, etc.)
Adding a `roles` hash can be sufficient to let the end-user customize that part.

Comment (FlorentFlament) [inline templates/default/aurora-security.erb:7]: Yes, customizing roles might be interesting. However, we would have to update the structure of user related information to add the groups each user belong to. This might be done in another commit.

Comment (pierrecdn) [inline templates/default/aurora-security.erb:7]: Fair enough, +1
